easybuild/easyblocks/p/pytorch.py

```diff
@@ -239,16 +239,10 @@ def configure_step(self):
         self.cfg.update('prebuildopts', ' '.join(unique_options) + ' ')
         self.cfg.update('preinstallopts', ' '.join(unique_options) + ' ')
 
-    def _set_cache_dir(self):
-        """Set $XDG_CACHE_HOME to avoid PyTorch defaulting to $HOME"""
-        cache_dir = os.path.join(self.tmpdir, '.cache')
-        # The path must exist!
-        mkdir(cache_dir, parents=True)
-        env.setvar('XDG_CACHE_HOME', cache_dir)
-
     def test_step(self):
         """Run unit tests"""
-        self._set_cache_dir()
```

Comment:
This change looks like a regression, I'm guessing `_set_cache_dir()` was created for a reason

Reply:
Yes, hadn't merged develop yet. Rebased and fixed and will add description now